src/server.py

```diff
@@ -0,0 +1,35 @@
+from ast import alias
+from xmlrpc.client import boolean
+import requests
+import json
+import time
+from fastapi import FastAPI, Request
+from fastapi.middleware.cors import CORSMiddleware
+from pydantic import BaseModel
+
+
+URL = 'http://127.0.0.1:8000'
+
+## RECEIVE DATA IN
+
```

Comment:
Delete this comment

Reply:
I'm keeping this comment so I understand what it's doing